app/db/queries.py

```diff
@@ -5,44 +5,74 @@
 
 
 def get_messages_that_contain_word(word: str) -> List[Message]:
-    pass
+    return list(Message.objects.select_related().filter(
+        text__icontains=word
+    ))
 
 
 def get_untitled_chats() -> List[Chat]:
-    pass
+    return list(Chat.objects.filter(title__icontains="Untitled"))
```

Comment:
Chat with the title `Hello, Untitled` should not be considered untitled. Please, check the statement one more time.